.external

```diff
@@ -1,2 +1,2 @@
-xamarin/monodroid:main@602aca98245744882a129206b79b5a5e093dae44
+xamarin/monodroid:thays_support_debug_server_n@4bc0f7acf04668e1fa1a6fca63ea68ca9b2d40ec
```

Comment:
We should merge the xamarin/monodroid side first, then update this line to monodroid/main.

Reply:
done.